config/initializers/countries.rb

```diff
@@ -200,3 +200,4 @@
 }.freeze
 
 COUNTRIES_BY_NAME = COUNTRIES.map(&:second)
+REVERSE_COUNTRIES_HASH = COUNTRIES.map(&:reverse).to_h
```

Comment:
I think it might be worth revisiting some of this naming.

`COUNTRIES_BY_NAME` could simply be `COUNTRY_NAMES`, as the 'by' suggests are more complex data structure than a simple list.

`REVERSE_COUNTRIES_HASH` could be `COUNTRY_NAMES_TO_ISO_CODES`, so its clearer what the constant contains without looking up its definition.

Reply:
All good points. I'll change them in the previous PR dealing with nationalities 👍